test_nlp.py

```diff
@@ -28,6 +28,13 @@
 )
 
 LOGGER = logging.getLogger(__name__)
+
+# create filehandler just for test errors for ease of human review
+error_log = logging.FileHandler("test_output.log", "w+")
+error_log.setLevel(logging.ERROR)
```

Comment:
I like having an erorr log file since that could be parsed by grep, vs code to find specific lines

Reply:
I was also thinking of having a level set from an env variable here

```
logging.basicConfig(
    level=logging.INFO,
```
level = dotenv['LOG_LEVEL']

kind of like what we had with discovery_config before (we can set the log level in the config and test_discovery / discovery would use it)

Reply:
Should we have separate log levels for log to file vs log to console?

Reply:
ahh, good point

maybe log to console defaults to INFO
and log to file defaults to ERROR

and when an env variable exists for LOG_LEVEL, both console and file are set to LOG_LEVEL ?

Reply:
I prefer less verbose logs in CI